class.jetpack-modules-registry.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+function jetpack_register_module( $slug, $args ) {
+	$jetpack_modules_registry = Jetpack_Modules_Registry::get_instance();
+	$jetpack_modules_registry->register( $slug, $args );
+}
+
+/**
+ * Class Jetpack_Modules_Registry
+ *
+ * Class for keeping the state of all modules in memory.
+ */
+class Jetpack_Modules_Registry {
+
+	private $modules = array();
+
+	private static $instance = null;
+
+	public static function get_instance() {
```

Comment:
as there's no constructor maybe the entire class could be `static`? not sure what's the best pattern here